pkg/kv/dist_sender.go

```diff
@@ -59,6 +60,36 @@ const (
 	defaultSenderConcurrency = 500
 )
 
+var (
+	// DistSender Metrics
+	metaTotalSends         = metric.Metadata{Name: "distsender.totalSends"}
+	metaLocalRPCs          = metric.Metadata{Name: "distsender.localRPCs"}
+	metaRetriesTimes       = metric.Metadata{Name: "distsender.retries"}
+	metaNotLeaseHolderErrs = metric.Metadata{Name: "distsender.notLeaseHolderErrs"}
+)
+
+// DistSenderMetrics used to metric some statics for debug.
+type DistSenderMetrics struct {
```

Comment:
the struct and its fields need to be exported as the registry will reflect over the struct, looking for metric fields.